settings.yml

```
@@ -17,3 +17,11 @@ subdomain_tracking:
   type: bool
   default: false
   description: Track visitors across main domain & subdomains, assuming discourse is on a subdomain.
+do_not_track:
+  type: bool
+  default: true
+  description: So tracking requests will not be sent if visitors do not wish to be tracked.
+disable_cookies:
+  type: bool
+  default: true
```

```suggestion
  default: false
```

Same as the previous comment, particular important here because existing consumers probably don't want to delete Matomo cookies soon after the theme component is updated.